sw/vendor/meson.build

```diff
@@ -48,3 +48,21 @@ sw_vendor_gtest = declare_dependency(
   dependencies: thread_dep,
 )
 # End googletest and googlemock build definitions
+
+# libclang_rt.profile build definitions
+sw_vendor_llvm_clang_rt_dir = 'llvm_clang_rt_profile/compiler-rt'
+sw_vendor_llvm_clang_rt_profile_dir = sw_vendor_llvm_clang_rt_dir / 'lib/profile'
+sw_vendor_llvm_clang_rt_inc_dir = include_directories(
+  sw_vendor_llvm_clang_rt_dir / 'include',
+  sw_vendor_llvm_clang_rt_profile_dir,
```

Comment:
Does it not work if this is not in the include directories?

Reply:
Thanks for catching this, it does. Done.